product_docs/docs/efm/4/efm_rel_notes/05_efm_45_rel_notes.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Version 4.4"
+---
+
+Enhancements, bug fixes, and other changes in EFM 4.4 include:
+
+| Type | Description |
+| ---- |------------ |
+| Security Fix | Failover Manager has been upgraded to use PostgreSQL JDBC Driver version 42.4.1 to address security vulnerability CVE-2022-31197. Note: Failover Manager is not impacted by the vulnerability, but no longer contains the impacted version of the library. |
+| Enhancement | When a primary node has been isolated from the cluster and rejoins, if no other node has been promoted, Failover Manager will restart the isolated database and resume monitoring as the primary node in the cluster. [Support tickets #937960, #968619, #81770, #81894] |
+| Bug Fix | Failover Manager will now properly choose a standby to promote in rare cases of LSN values being compared incorrectly. [Support ticket #83186] |
+| Bug Fix | Failover Manager now handles extra whitespace in properties file when running the `efm` command for cluster status. |
+| Bug Fix | Failover Manager will no longer fail to promote in rare cases where the primary agent is running slowly. [Support ticket #79847] |
+| Bug Fix | Failover Manager properly follows the promotable=false value of a standby that is restarted after being made promotable manually through the `efm` command. [Support ticket #83192] |
+| Bug Fix | When using physical replication slots, Failover Manager will now drop and recreate slots on standbys when advancing the LSN values on them if they have an `xmin` value set. [Support ticket #83910] |
+| Bug Fix | Systemd status will now show "inactive" status instead of "failed" after being used to stop a Failover Manager agent. |
```

Comment:
There is no support ticket for this one. It's something that came up internally.